airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -48,6 +48,10 @@ definitions:
     $ref: "#/definitions/retriever"
     record_selector:
       $ref: "#/definitions/selector_increment"
+      # Some streams, e.g. Files stream, can return data without cursor field(update_time) in it, which leads to value error for cursor.
```

Comment:
As discussed synchronously, we should probably be able to sync them which probably means a change in the ConcurrentCursor in the CDK. It would be nice to understand what is the consequence in the destination though. Let's ask #dev-move on Slack!